### DISCUSSION TITLE
Slack Notification GitHub Action Failing Due to Missing Webhook URL

### DISCUSSION THREAD

**Discussion by michangelis** [zauberzeug/nicegui#1068] [Q&A] [answered]
### Question

Hello,

I encountered an issue while contributing to the repository. It seems the GitHub action Gamesight/slack-workflow-status@master is failing due to a missing Slack webhook URL.

Here is the error message I received:

```
Error: Error: Input required and not supplied: slack_webhook_url
Error: Input required and not supplied: slack_webhook_url
```

As an external contributor, I do not have the necessary permissions to add the SLACK_WEBHOOK_URL secret to the repository's settings. The secret appears to be necessary for the GitHub action to send notifications to Slack.

Could you please look into this issue? I believe that adding the SLACK_WEBHOOK_URL secret to the repository's settings should resolve it.

Please let me know if there's anything else I can provide to assist in resolving this.

**Comment by rodja** [comment]
Maybe the slack notify action should only be executed if the SLACK_WEBHOOK_URL environment variable is set.

**Comment by rodja** [comment, marked as answer]
The issue #1084 has been fixed by a pull-request from @michangelis.
